tests/components/accuweather/test_sensor.py

```diff
@@ -47,7 +47,7 @@ async def test_sensor_without_forecast(hass):
     assert state.attributes.get(ATTR_ATTRIBUTION) == ATTRIBUTION
     assert state.attributes.get(ATTR_ICON) == "mdi:weather-fog"
     assert state.attributes.get(ATTR_UNIT_OF_MEASUREMENT) == LENGTH_METERS
-    assert state.attributes.get(ATTR_STATE_CLASS) == STATE_CLASS_MEASUREMENT
+    assert state.attributes.get(ATTR_STATE_CLASS) == SensorStateClass.MEASUREMENT
```

Comment:
I think it might make sense to use "is" rather than "==" now that we are using enums

Reply:
Marked as draft until https://github.com/home-assistant/core/pull/61989#discussion_r770296057 is finalised